src/api/app/websocket.ts

```diff
@@ -92,11 +100,15 @@ export class AppWebsocket implements AppClient {
   private constructor(
     client: WsClient,
     appInfo: AppInfo,
-    defaultTimeout?: number
+    defaultTimeout?: number,
+    callZomeTransform?: CallZomeTransform
```

Comment:
Which are users more likely to set? I'd say the `callZomeTransform` is probably more common than overriding the timeout so maybe that comes before the `defaultTimeout` in the params list?

Reply:
Probably not so important either way

Reply:
Good question, I was assuming the opposite but you might be right. Holochain has an internal timeout of 60 seconds is that right? So the only sensible reason to change the timeout here would be to make it shorter which I'm not sure many people would want to do.